pkg/utils/command.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"os/exec"
+	"reflect"
+	"strings"
+)
+
+// ExecTag stand for the path of executable file in command.
+// If we want this util works ,
+// we must add Exec in the struct and use NewExec() to initialize it,
+// because the default way to initialize Exec means None in code.
+const ExecTag = "exec"
+
+// SubCommandTag stand for the sub command in common command.
+// We can use it in struct fields as a tag.
+// Just like MatchExtension below
+// type Iptables Struct {
+// 	Exec
+//	MatchExtension string `sub_command:""`
+// }
+// Field with SubcommandTag needs to be a struct with Exec.
+const SubCommandTag = "sub_command"
+
+// ParaTag stand for parameters in command.
+// We can use it in struct fields as a tag.
+// Just like Port below
+// type Iptables Struct {
+// 	Exec
+//	Port string `para:"-p"`
+// }
+// If the field is not string type or []string type , it will be skipped.
+// If the tag value like "-p" is empty string ,
+// the para will just add the field value into the command just as some single value parameter in command.
+// If the value of field is empty string or empty string slice or empty slice, the field and tag will all be skipped.
+const ParaTag = "para"
+
+type Exec struct {
+	option string
+}
+
+func NewExec() Exec {
```

Comment:
Add some comments to describe when to use it and why need to use it.

Reply:
Fixed.